src/plugins/rgbcolor/README.md

```diff
@@ -0,0 +1,86 @@
+- infos = Information about the rgbcolor plugin is in keys below
+- infos/author = Philipp Gackstatter <philipp.gackstatter@student.tuwien.ac.at>
+- infos/licence = BSD
+- infos/needs =
+- infos/provides =
+- infos/recommends =
+- infos/placements = postgetstorage presetstorage
+- infos/status = maintained unittest nodep
+- infos/metadata = check/rgbcolor
+- infos/description = Validation of rgbcolors
+
+## Introduction
+
+tba
+
+## Usage
+
+- add your plugin in `src/plugins/README.md`
+- and rewrite the rest of this `README.md` to give a great
+  explanation of what your plugin does
+
+## Dependencies
+
+None.
+
+## Examples
+
+```sh
+# Mount a config file with the rgbcolor plugin
+sudo kdb mount color.ecf user/tests/color dump rgbcolor
+
+# Suceeds, since the value is a valid rgbcolor. Quotes are important!
+kdb set user/tests/color/hex "#fff"
+
+# Check the user/tests/color/hex key for validity
+kdb setmeta user/tests/color/hex check/rgbcolor any
+
+# Suceeds, since the value is a valid rgbcolor. Quotes are important!
+kdb set user/tests/color/hex "#a1C2b3"
+
+# Suceeds, since the value is a valid RGBA rgbcolor. Quotes are important!
+kdb set user/tests/color/hex "#aabbccdd"
+
+# Colors are normalized to bytes
+kdb get user/tests/color/hex
+#> \xaa\xbb\xcc\xdd
```

Comment:
@markus2330 
Is it okay to return the bytes like that? I suppose this is good for any program that wants to read the values, but it's not so nice for the enduser.

Reply:
I think it is a bit weird to parse this string inside a C program. 

IMO we should either convert to decimal (i.e. `2864434397` in your case), to ensure optimal compatibility with the highlevel API. Or just convert to the standard `#aabbccdd` representation, as extending the highlevel API is not really hard.

Reply:
I agree with @kodebach. I would prefer the decimal variant.